webpack.config.js

```diff
@@ -0,0 +1,55 @@
+const path = require('path');
```

Comment:
this filename should not be webpack config, cause when you call just webpack it looks for default config with exactly this name. Please change it to webpack.server.config.js